openapi/platform/platform.yaml

```diff
@@ -1802,7 +1802,79 @@ paths:
           required: true
       security:
         - OAuth2:
-            - automation:read
+            - 'automation:read'
+      tags:
+        - Automations
+  /automations:
+    parameters: []
+    get:
+      summary: List Automations
+      responses:
+        '200':
+          description: OK
+          content:
+            application/vnd.api+json:
+              schema:
+                type: array
+                description: ''
+                minItems: 1
+                uniqueItems: true
+                x-examples:
+                  example-1:
+                    - type: automations
+                      id: Automation-91cb048c-c498-43ba-ab99-2ca2d27d2c46
+                      attributes:
+                        name: Example Automation
+                items:
+                  type: object
+                  properties:
+                    type:
+                      type: string
+                      minLength: 1
+                    id:
+                      type: string
+                      minLength: 1
+                    attributes:
+                      type: object
+                      required:
+                        - name
+                      properties:
+                        name:
+                          type: string
+                          minLength: 1
+                  required:
+                    - type
+                    - id
+                    - attributes
+              examples:
+                example-1:
+                  value:
+                    data:
+                      - id: Automation-abcd1234-12ab-12ab-12ab-12ab12ab12ab
+                        type: automations
+                        attributes:
+                          name: Example Automation
```

Comment:
For the list operation, we should return the `status`, `tags`, and `trigger` as well. 

I wouldn't add the `type` field just yet. My understanding is that it will eventually be going away once data passing is fully developed.